docs/working.md

```diff
@@ -11,10 +33,23 @@ Currently we provide `apollo-typescript-starter` only, and we'll be supporting m
 #### [apollo-typescript-starter](https://github.com/aerogear/graphback/tree/master/templates/apollo-starter-ts)
 Production ready typescript implementation of GraphQL server using apollo-express framework. Uses knex.js library for database access.
 
+## Generated source code
+
+Generated server side code will consist of the template together with additional elements:
+
+`./model` folder will contain input schema
+`./resolvers` folder will contain generated and custom GraphQL resolvers for the project.
+
 ## Databases
 Graphback allows you to configure your database according to your needs. Graphback provides support to multiple databases(MySQL, MongoDB etc).
 Currently, we are supporting `PostgreSQL`.
 
+
+## GraphQL Modules
```

Comment:
This is to be done

Reply:
Yes.. we can mention that in documentation to give overview.